### PR TITLE
Add darwin CI job

### PR DESCRIPTION
**Pull request by at-wat**
Fix unstable test on OSX environment

### Reference issue
Part of #174

### CODE REVIEW

**Comment by lherman-cs** [comment]
@at-wat It looks like the instability mainly comes from the OSX. I wonder if we can configure the system to increase the stability.

**Comment by at-wat** [comment]
## ticker accuracy
interval | count | Linux | OSX
--- | --- | --- | ---
100µs | 10 | 1.07ms | 4.75ms
1ms | 10 | 14.00ms | 21.86ms
10ms | 10 | 100.15ms | 131.97ms
100ms | 10 | 1.00015s | 1.10824s

Timer on OSX seems just awful.

**Comment by at-wat** [comment]
@lherman-cs if it looks good to you, I'll update branch protection rule and merge.

**Comment by lherman-cs** [comment]
The weird thing is that it is fine on my local machine.

interval | count | Linux | OSX | Local Machine (also using Catalina)
--- | --- | --- | --- | ---
100µs | 10 | 1.07ms | 4.75ms | 1.03ms
1ms | 10 | 14.00ms | 21.86ms | 10.01ms
10ms | 10 | 100.15ms | 131.97ms | 102.31ms
100ms | 10 | 1.00015s | 1.10824s | 1.0048s

**Comment by lherman-cs** [comment]
> @lherman-cs if it looks good to you, I'll update branch protection rule and merge.

What update are you referring?

**Comment by at-wat** [comment]
So, it might be a GitHub Action environment specific problem? (high load)

**Comment by lherman-cs** [comment]
> So, it might be a GitHub Action environment specific problem? (high load)

Yea, that's what I'm leaning toward to now.

**Comment by at-wat** [comment]
image
Update to make new jobs `Required` and remove old jobs from required status.

**Comment by lherman-cs** [comment]
> image
> Update to make new jobs `Required` and remove old jobs from required status.

Yes, please. Thanks!

File: pkg/io/video/throttle_test.go [hunk 1]
```diff
@@ -9,25 +9,25 @@ import (
 func TestThrottle(t *testing.T) {
 	img := image.NewRGBA(image.Rect(0, 0, 640, 480))
 
-	ticker := time.NewTicker(time.Millisecond)
+	ticker := time.NewTicker(20 * time.Millisecond)
```
Comment (at-wat) [inline pkg/io/video/throttle_test.go:12]: New parameter is way realistic than before.

**Pull request merged**